src/java/org/apache/cassandra/cql3/CQL3Type.java

```diff
@@ -566,6 +567,21 @@ public String toCQLLiteral(ByteBuffer buffer)
             return sb.toString();
         }
 
+        @Override
```

Comment:
This should have been added when Vector was added correct?

Reply:
Yes, they should. Without proper hashcode/equals we can end up seeing messages such as `Masking function mask_default() return type is vector<int, 2>. This is different to the type of the masked column r2 of type vector<int, 2>.`